tests/test-xbackstage/resources.k

```diff
@@ -0,0 +1,379 @@
+import models.io.upbound.platform.aws.v1alpha1 as platformawsv1alpha1
+import models.io.crossplane.helm.v1beta1 as helmv1beta1
+
+_xeks = platformawsv1alpha1.XEKS {
```

Comment:
Any particular reason to use underscore here?

Reply:
So i can override/add additional fields it in the tests